source/reference/limitations.txt

```diff
@@ -135,9 +133,8 @@ Sharded Clusters
   syncing.
 - The shard key cannot be modified using :dbcommand:`reshardCollection`
   during syncing.
-- The maximum number of :ref:`shard key indexes
-  <sharding-shard-key-indexes>` is one lower than normal, 63 instead of
-  64.
+- The maximum number of indexes per sharded collection is 63, which is
+  one less than the :ref:`normal limit <limit-index-size>` of 64. 
```

Comment:
Per Sarah, consider "default" or "typical" rather than "normal". Understood not your original content :)

Reply:
Done! Thanks for the feedback!